packages/react/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+
+import { StepperProps } from './Stepper.types';
+
+import clsx from 'clsx';
+import { getStepperUtilityClass, stepperClasses } from './Stepper.classes';
+
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+
+import { styled, useThemeProps } from '@mui/material/styles';
+
+type StepperOwnerState = {
+  classes?: StepperProps['classes'];
+  orientation?: 'horizontal' | 'vertical';
+  activeStep?: StepperProps['activeStep'];
+  size?: StepperProps['size'];
+  position?: 'default' | 'bottom';
+  size24?: boolean;
+};
+
+const useUtilityClasses = (ownerState: StepperOwnerState) => {
+  const { classes, orientation, size24 } = ownerState;
+
+  const slots = {
+    root: ['root', orientation, size24 && 'size24'],
+    container: ['container'],
+    activeStep: ['activeStep'],
+    size: ['size'],
+  };
+
+  return composeClasses(slots, getStepperUtilityClass, classes);
+};
+
+const StepperRoot = styled('div', {
+  name: 'ESStepperContainer',
```

Comment:
```suggestion
  name: 'ESStepper',
```
И остальные поправить в этом файле